packages/components/src/components/domainproperties/list/ListPropertiesPanel.tsx

```diff
@@ -125,16 +134,16 @@ class ListPropertiesPanelImpl extends React.PureComponent<Props, State> {
         this.onChange(id, value);
     };
 
-    applyAdvancedProperties = (advancedSettingsForm: AdvancedSettingsForm) : void => {
-        const { model, onChange } = this.props;
+    applyAdvancedProperties = (advancedSettingsForm: AdvancedSettingsForm) => {
+        const { model } = this.props;
         const newModel = model.merge(advancedSettingsForm) as ListModel;
-        onChange(newModel);
+        this.setIsValid(newModel);
     };
 
     render() {
         const { panelStatus, collapsible, controlledCollapse, model, useTheme, successBsStyle } = this.props;
         const { isValid } = this.state;
-        const { collapsed } = this.context;
+        const { collapsed } = this.context || DEFAULT_CONTEXT;
```

Comment:
Should this use default value syntax? I think there is some subtle difference here that I can't quite put my finger on.
```suggestion
        const { collapsed = DEFAULT_CONTEXT.collapsed } = this.context;
```

Reply:
This DEFAULT_CONTEXT is only needed for the jest tests. Since this component uses the DomainPropertiesPanelContext, the jest tests got mad about the this.context not being defined. I'm hoping to refactor these DomainPropertiesPanelContext usages to maybe use an HOC or something that would allow for removing this DEFAULT_CONTEXT.